03/styles/mobile.css

```diff
@@ -0,0 +1,16 @@
+body {
+    grid-template-columns: 1fr;
+    grid-template-rows: 1fr 1fr 1fr;
+    grid-template-areas: 
+    "P"
+    "T"
+    "V";
+}
+
+p {
+    max-height: 90%;
+    text-align: center;
+    overflow-y: scroll;
```

Comment:
Nie powiedziałbym, że scroll to dobre rozwiązanie. 
Nie dość, że mamy jeden scroll domyślny, to wew. jeszcze dodatkowy.
Raczej bym z tego zrezygnował ;)